integration/kubernetes/runtimeclass_workloads/pod-bad-health.yaml

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+apiVersion: v1
+kind: Pod
+metadata:
+  name: badhctest
+spec:
+  runtimeClassName: kata
+  containers:
+  - name: test
+    image: mhausenblas/simpleservice:0.5.0
+    env:
+    - name: HEALTH_MIN
```

Comment:
I'm presuming these HEALTH values are chosen/set to create a desired effect (bad health timeout/fail?). can we have some comments here explaining that, otherwise the numbers are somewhat opaque (and we'll forget over time what we set and why etc.).